src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerTestsBase.cs

```diff
@@ -21,7 +21,8 @@ public class InvitationsControllerTestsBase : TestBase
         private const string FunctionalRoleCode = "FRC";
         protected const string InvitationLocation = "InvitationLocation";
         private const string AzureOid = "47ff6258-0906-4849-add8-aada76ee0b0d";
-        protected readonly int InitialInvitationId = TestFactory.Instance.KnownTestData.InvitationIds.First();
+        protected readonly int InitialMdpInvitationId = TestFactory.Instance.KnownTestData.InvitationIds.First();
```

Comment:
Too fragile to use .First and .Last here ..  Split TestFactory.Instance.KnownTestData.InvitationIds into 2 sep lists